jdaviz/configs/imviz/wcs_utils.py

```diff
@@ -440,12 +440,10 @@ def _prepare_rotated_nddata(real_image_shape, wcs, rotation_angle, refdata_shape
 
     # create a fake NDData (we use arange so data boundaries show up in Imviz
     # if it ever is accidentally exposed) with the rotated GWCS:
-    sequential_data = np.arange(
-        np.prod(refdata_shape), dtype=np.int8
-    ).reshape(refdata_shape)
+    placeholder_data = np.nan * np.ones(refdata_shape)

```

Comment:
This was discussed in an earlier PR that we merged, but then I botched a rebase, so I'm fixing that mistake in this PR. Since this was approved before, I'm just reinserting it.

There are two main benefits if the "WCS-only reference data" _have no data_: 
1) when a WCS-only layer is created, it is visible for a few milliseconds, before the visibility is set to `False`. If the data are `nan`, they are never rendered, and the viewer does less crazy flashing during rotation. 
2) If you somehow do math on the WCS-only layer's pixels, the result will never appear to be legitimate. This should not be easy to do, given the checks we have in place, but switching to `nan` makes it much easier to debug if we accidentally do operations on the WCS-only data.

Reply:
Yes, I think the benefit not having the weird flashing is good. Thanks for the clarification!